### ISSUE TITLE
Suggestion: Eye Color Sliders

### ISSUE THREAD

**Issue by nearlyNonexistent** [TheFlyingMinotaur/CharacterCreatorRelease#3]

**Comment by Aegide** [comment]
Hello !

It seems like @cyaggi added that suggestion to [his fork](https://github.com/cyaggi/CharacterCreatorRelease)!

**Comment by cyaggi** [comment]
Yes I did. I added an eye slider (though due to how the images are, the eyes are the same color as beards), and made all the sliders into color choosers to make it easier.
